msrestazure/azure_operation.py

```diff
@@ -59,6 +59,30 @@ def succeeded(status):
     return str(status).lower() in SUCCEEDED
 
 
+def _validate(url):
+    """Validate is valid url.
+
+    :param str url: Polling URL extracted from response header.
+    :returns: URL if valid.
+    :raises: ValueError if URL has not scheme or host.
+    """
+    if url is None:
+        return None
+    parsed = urlparse(url)
+    if not parsed.scheme or not parsed.netloc:
+        raise ValueError("Invalid URL header")
+    return url
```

Comment:
If the function is checking whether a URL is valid or not, should it return `True`/`False` instead since you are asking a question? Or perhaps return nothing and simply raise an exception if the validation failed?